tests/conftest.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""Test Config
+"""
+
+import sys
+import pytest
+import shutil
+from os import path, mkdir
+
+sys.path.insert(1, path.abspath(path.join(path.dirname(__file__), path.pardir)))
+
+@pytest.fixture(scope="session")
+def nwTemp():
```

Comment:
Why not https://docs.python.org/3.8/library/tempfile.html ?

Reply:
Because we don't want the folder structure destroyed after the test. We want to clear it before instead.